src/Guide/Api/Methods.hs

```diff
@@ -58,6 +60,26 @@ createCategory db title' group' = do
       -- TODO addEdit edit
       return catId
 
+-- | Edit categoty's note.
+setCategoryNotes :: DB -> Uid Category -> Text -> Handler NoContent
+setCategoryNotes db catId note = uncache db (CacheCategoryNotes catId) $ do
+    (_edit, _newCategory) <- dbUpdate db (SetCategoryNotes catId note)
+    pure NoContent
+
+-- | Edit category`s info (title, group, status, sections (pro/con, ecosystem, note))
+setCategoryInfo :: DB -> Uid Category -> CCategoryInfoEdit -> Handler NoContent
+setCategoryInfo db catId CCategoryInfoEdit {..} =
+    uncache db (CacheCategoryInfo catId) $ do
+        _ <- dbUpdate db $ (\(H cTitle) -> SetCategoryTitle catId cTitle) ccieTitle
```

Comment:
I would recommend adding something like `unH` and then you can write these in a nicer way:

```haskell
_ <- dbUpdate db $ SetCategoryTitle catId (unH ccieTitle)
```